components/hab/src/main.rs

```diff
@@ -119,6 +143,19 @@ fn start(ui: &mut UI) -> Result<()> {
                 _ => unreachable!(),
             }
         }
+        ("config", Some(m)) => {
+            match m.subcommand() {
+                ("apply", Some(m)) => sub_svc_set(m)?,
```

Comment:
Should we change the name of this subcommand to `set`, with an alias of `apply`? That way we can start weaning folks over to the command with the better name.

Also, why `sub_svc_set` instead of `sub_config_set`?